client/my-sites/plans-v2/details.tsx

```diff
@@ -75,7 +75,10 @@ const DetailsPage = ( { duration, productSlug, rootUrl, header }: DetailsPagePro
 			<HeaderCake onClick={ backButton }>
 				{ isBundle ? translate( 'Bundle Options' ) : translate( 'Product Options' ) }
 			</HeaderCake>
-			<FormattedHeader headerText="Great choice! Now select a backup option:" brandFont />
+			<FormattedHeader
+				headerText={ translate( 'Great choice! Now select a Backup option:' ) }
```

Comment:
When this code was originally written, this wasn't a translation as this header needs to change for options that are not backup (such as Security), and this needs to not be sent to translation until that happens.

The two options I know of are Backup and Security.

Reply:
Got it, I'll add the logic for that.